components/ResourceCreation/PatientCreation.tsx

```diff
@@ -176,8 +230,32 @@ function PatientCreation({
                     >
                       <Trash />
                     </Button>
+                    <Button data-testid="calculate-button" onClick={() => clickCalculateButton(id)} color="gray">
+                      Calculate
+                    </Button>
+                    {currentPatients[id].measureReport !== undefined && (
+                      <Button
+                        data-testid="toggle-show-calculation-button"
+                        onClick={() => setShowCalculation(o => !o)}
+                        color="gray"
+                      >
+                        {showCalculation === true ? `Hide Results` : `Show Results`}
```

Comment:
I think a button name like "Show Logic Highlighting"/"Hide Logic Highlighting" or something makes more sense so the user knows what to expect?